lib/src/signed_video_h26x_sign.c

```diff
@@ -641,11 +640,10 @@ signed_video_get_nalu_to_prepend(signed_video_t *self,
     } else {
       nalu_to_prepend->prepend_instruction = SIGNED_VIDEO_PREPEND_NOTHING;
```

Comment:
You should should reset the input `nalu_to_prepend` before calling get_latest_sei

Reply:
If I call it before the get_latest_sei I would also reset the |sei_size|.
Should I reset it  before we get the |sei_size|?

Reply:
Yes. When you have outputs returned through a pointer like here you need to reset the content before doing anything. It is good practice to do that. For example, if the pointer garbage data when passed in and some parts are not changed the user may read the data in this pointer and make conclusions based on garbage.